app/src/main/java/com/example/mobiledger/presentation/home/HomeViewModel.kt

```diff
@@ -26,7 +30,28 @@ class HomeViewModel(
     val userNameLiveData: LiveData<String> get() = _userNameLiveData
     private val _userNameLiveData: MutableLiveData<String> = MutableLiveData()
 
-    fun getUserName() {
+    val homeViewItemListLiveData: LiveData<Event<MutableList<HomeViewItem>>> get() = _homeViewItemListLiveData
+    private val _homeViewItemListLiveData: MutableLiveData<Event<MutableList<HomeViewItem>>> = MutableLiveData()
+
+    val monthNameLiveData: LiveData<String> get() = _monthNameLiveData
+    private val _monthNameLiveData: MutableLiveData<String> = MutableLiveData()
+
+    private val _isLoading: MutableLiveData<Boolean> = MutableLiveData(false)
+    val isLoading: LiveData<Boolean> get() = _isLoading
+
+    private val _errorLiveData: MutableLiveData<Event<ViewError>> = MutableLiveData()
+    val errorLiveData: LiveData<Event<ViewError>> = _errorLiveData
+
+    private var monthCount = 0
+
+    fun getHomeData(){
+        _isLoading.value = true
```

Comment:
If you are setting isLoading true here and calling 3 fun in parallel, so whichever will be completed first will set its value to false even if other 2 are still running. Is that correct?